<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentMain</class>
 <widget class="QDialog" name="studentMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QWidget" name="studentMainLogin">
    <widget class="QLabel" name="welcomeLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Welcome</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>*name*</string>
     </property>
    </widget>
    <widget class="QLabel" name="upcomingTasksLabel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Upcoming Tasks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="allTasksLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>All Tasks / Completed Tasks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>110</y>
       <width>191</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>611</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="changeEmailButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changePasswordButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeNameButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeSurnameButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Surname</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
